roles/kubernetes/master/templates/kubeadm-config.v1alpha3.yaml.j2

```diff
@@ -191,6 +191,14 @@ apiServerExtraVolumes:
   mountPath: {{ volume.mountPath }}
   writable: {{ volume.writable | default(false)}}
 {% endfor %}
+{% if ssl_ca_dirs|length %}
+{% for dir in ssl_ca_dirs %}
+- name: {{ dir | regex_replace('^/(.*)$', '\\1' ) | regex_replace('/', '-') }}
+  hostPath: {{ dir }}
+  mountPath: {{ dir }}
+  readOnly: true
```

Comment:
`readOnly` is only supported in `v1beta1`. For the other versions, use `writable`

Reply:
Thanks i updated the patch